# Base
axis_base_linear_for: 5
axis_base_linear_bac: 2
axis_base_angular: 0
scale_base_linear: 0.5
scale_base_linear_turbo: 1.0
scale_base_angular: 0.75
# swtich torbo
torbo_button: 1

# Camera
axis_camera_angular_yaw: 3
axis_camera_angular_pitch: 4
scale_camera_angular: 0.5
